libs/api/domains/application/src/lib/application.model.ts

```diff
@@ -9,37 +9,37 @@ registerEnumType(ApplicationResponseDtoTypeIdEnum, {
 @ObjectType()
 export class Application {
   @Field(() => ID)
-  id: string
+  id!: string
 
   @Field(() => Date)
-  created: Date
+  created!: Date
 
   @Field(() => Date)
-  modified: Date
+  modified!: Date
 
   @Field(() => String)
-  applicant: string
+  applicant!: string
 
-  @Field(() => String)
-  assignee: string
+  @Field(() => [String], { nullable: true })
+  assignees!: string[]
```

Comment:
just wondering, if nullable is true, do you still need to mark assignees with `!` ?